src/cart/src/cart/crt_group.h

```diff
@@ -235,10 +233,10 @@ struct crt_uri_item {
 	d_rank_t	ui_rank;
 
 	/* reference count */
-	uint32_t	ui_ref;
+	ATOMIC uint32_t	ui_ref;
 
 	/* flag indicating whether initialized */
-	uint32_t	ui_initialized;
+	uint32_t	ui_initialized:1;
 
 	/* mutex for protection of ui_ref */
 	pthread_mutex_t ui_mutex;
```

Comment:
Can you remove this mutex now?

Reply:
No, it's used for other locking also.

Reply:
Ah, then it's the comment that's wrong.

Reply:
Agree. Comment is wrong. I didn't realize this.  Anyway I will remove this mutex in next patch I'm preparing now.